app/src/features/FhirAttributePanel/AttributeInputGroup.tsx

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+
+import { Icon } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import {
+  Button,
+  ListItemText,
+  makeStyles,
+  Menu,
+  MenuItem,
+  Paper,
+  Typography,
+} from "@material-ui/core";
+import { Add } from "@material-ui/icons";
+import clsx from "clsx";
+import {
+  bindMenu,
+  bindTrigger,
+  usePopupState,
+} from "material-ui-popup-state/hooks";
+import { useTranslation } from "react-i18next";
+
+import {
+  useApiInputGroupsDestroyMutation,
+  useApiInputsCreateMutation,
+  useApiInputsListQuery,
+} from "services/api/endpoints";
+import { InputGroup } from "services/api/generated/api.generated";
+
+import Input from "./Inputs";
+
+type AttributeInputGroupProps = {
+  inputGroup: InputGroup;
+};
+
+const useStyles = makeStyles((theme) => ({
+  inputGroups: {
+    padding: theme.spacing(2),
+    backgroundColor: theme.palette.background.default,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+  },
+  button: {
+    textTransform: "none",
+    marginTop: theme.spacing(1),
+  },
+  buttonCondition: {
+    marginTop: theme.spacing(3),
+  },
+  buttonDelete: {
+    alignSelf: "flex-end",
+  },
+  deleteIcon: {
+    fill: theme.palette.getContrastText(theme.palette.background.paper),
+  },
+  menu: {
+    marginTop: theme.spacing(1),
+  },
+}));
+
+const AttributeInputGroup = ({
+  inputGroup,
+}: AttributeInputGroupProps): JSX.Element => {
+  const { t } = useTranslation();
+
+  const popupState = usePopupState({
+    variant: "popover",
+    popupId: "popup",
+  });
+
+  const [deleteInputGroups] = useApiInputGroupsDestroyMutation();
+  const [createInput] = useApiInputsCreateMutation();
+  const { data: inputs } = useApiInputsListQuery({ inputGroup: inputGroup.id });
+
+  const handleDeleteInputGroup = async () => {
+    if (inputGroup) {
```

Comment:
inputGroup should always be defined as it is stated in `AttributeInputGroupProps` type definition